unittest/gunit/keyring_vault/vault_parser-t.cc

```diff
@@ -209,6 +210,18 @@ namespace keyring__vault_parser_unittest
     EXPECT_TRUE(vault_parser.parse_key_data(payload, &key));
   }
 
+  TEST_F(Vault_parser_test, ParsePayloadThatsGarbage)
+  {
+    uchar garbage[101000];
```

Comment:
This shouldn't be addressed, just out of curiosity, why 101000? :)

Reply:
No particular reason :)